src/utils_internal.rs

```diff
@@ -198,8 +198,10 @@ pub async fn build_tcp_stream(
     Ok(StreamHandle::from_stream(stream))
 }
 
+#[allow(dead_code)]
```

Comment:
FYI, you're going to have a conflict with #12 here

Reply:
Let's wait till #12 is in, and I can fix up then. No idea if additional changes will land before it does.